challenges-webapp/src/main/webapp/vue-app/engagement-center/components/achievements/EditRealizationDrawer.vue

```diff
@@ -0,0 +1,194 @@
+<!--
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2022 Meeds Association
+contact@meeds.io
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation; either
+version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+Lesser General Public License for more details.
+You should have received a copy of the GNU Lesser General Public License
+along with this program; if not, write to the Free Software Foundation,
+Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+-->
+<template>
+  <exo-drawer
+    ref="editRealizationDrawer"
+    class="editRealizationDrawer"
+    :right="!$vuetify.rtl"
+    @closed="close"
+    eager>
+    <template slot="title">
+      <span class="pb-2"> {{ $t('realization.label.editRealization') }} </span>
+    </template>
+    <template slot="content">
+      <v-card-text>
+        <div class="mb-2">
+          <span class="subtitle-1 labels"> {{ $t('challenges.label.reward') }}</span>
+          <v-text-field
+            v-model="points"
+            :label="$t('realization.label.points')"
+            :rules="[rules.value]"
+            :placeholder="$t('realization.label.points')"
+            class="pointsRealization"
+            type="number"
+            outlined
+            required />
+        </div>
+        <span class="subtitle-1 labels"> {{ $t('realization.label.actionLabel') }}</span>
+        <div class="actionLabel">
+          <v-textarea
+            v-model="actionLabel"
+            :placeholder="$t('realization.label.actionLabel')"
+            :rules="[rules.length]"
+            name="actionLabel"
+            class="pl-0 pt-0 actionLabel"
+            auto-grow
+            required
+            autofocus
+            rows="5" />
+          <span class="tooManyChars mt-n2">
+            {{ charsCount }}{{ maxLength > -1 ? ' / ' + maxLength : '' }}
+            <i class="uiIconMessageLength"></i>
+          </span>
+        </div>
+        <div class="mt-2">
+          <span class="subtitle-1 labels"> {{ $t('realization.label.program') }}</span>
+          <v-flex>
+            <v-autocomplete
+              ref="selectAutoComplete"
+              v-model="program"
+              :label="labels.label"
+              :placeholder="labels.placeholder"
+              :items="domains"
+              :loading="loadingSuggestions"
```

Comment:
use this variable and affect its value when loading is in progress (it seems not changed at all)